action/build_get.go

```diff
@@ -87,15 +86,12 @@ DOCUMENTATION:
 // input and create the object used to
 // capture a list of builds.
 func buildGet(c *cli.Context) error {
-	// create a vela client
-	client, err := vela.NewClient(c.String("addr"), nil)
+	// parse the Vela client from the context
+	client, err := client.Parse(c)
```

Comment:
Each command will now use `go-vela/cli/internal/client.Parse()` to parse the client from the CLI context.

This gives us a few benefits like less LOC, makes all commands follow the same code execution path and makes any future changes easier to implement because the code is in a single package.